docs/queries/derivativeShd.md

```diff
@@ -0,0 +1,54 @@
+# Shade Derivative Examples
```

Comment:
Lets put a note on here (and in the comments of all functions) that this is still in development and on testnet only, because once we go to main it's going to automatically put out a release. I don't want our public users to think this is finalized and usable.

Reply:
Done